bin/lib/zopen-build

```diff
@@ -917,7 +921,7 @@ resolveCommands()
     unset ZOPEN_MAKE_CMD
   fi
 
-  if [ "${ZOPEN_CHECK}x" != "skipx" ] && [ ! -z "$(command -V ${ZOPEN_CHECK})" ] && ! ${skipcheck}; then
+  if [ "${ZOPEN_CHECK}x" != "skipx" ] && ! ${skipcheck}; then
```

Comment:
why is this check that the command exists being removed?

Reply:
This is to enable zoslib, since the check command may be created during the build process and this is checked before the build is run